python/packages/sdk-schema/pyproject.toml

```diff
@@ -1,14 +1,40 @@
 [build-system]
-requires = ["setuptools>=65.6.3", "wheel"]
 build-backend = "setuptools.build_meta"
+requires = [
```

Comment:
This changes seem unrelated to this PR.

Let's keep this PR only about CI publication setup, and not mixin other work here

Reply:
I believe this change was necessary to pass `pyproject-fmt`'s validation after adding the build deps necessary to publish builds of this module, it seems to insist on an order to how the metadata is listed.

Reply:
Yeah, I just ran it again, it's a change made by `pyproject-fmt`

Reply:
Happy to remove it, just don't think this PR will pass CI validation without it

Reply:
If it's needed in context of this PR, let's keep it. I blindly assumed this change relates to SDK Schema generation which is not in context here